security/.htaccess

```diff
@@ -13,18 +13,24 @@ RewriteRule ^v3-unstable$ https://w3c-ccg.github.io/security-vocab/contexts/secu
 # https://w3id.org/security/data-integrity
 
 RewriteRule ^data-integrity$ https://www.w3.org/TR/vc-data-integrity/ [R=302,L]
-RewriteRule ^data-integrity/v1$ https://www.w3.org/ns/data-integrity/v1 [R=302,L]
+RewriteRule ^data-integrity/v1$ https://w3c.github.io/vc-data-integrity/contexts/data-integrity/v1 [R=302,L]
```

Comment:
These are going to come through as `application/octet-stream`.  Would be nice if the source repos used `.jsonld` for the contexts so github serves them as `application/ld+json`.  Similar to the other contexts in this file.

Reply:
hrm, yes, good point. Does Github pages allow you to set media types on files?

Reply:
That github pages feature hasn't existed in the past.  I suspect it still does not.  Target file extensions fix the issue, people are probably not linking direct to those files, and it helps text editors know what syntax to use.

Reply:
I think pages handles symlinks too, but that makes the target dirs kind of a mess.